lisa/sut_orchestrator/azure/features.py

```diff
@@ -1456,13 +1456,25 @@ def create_setting(
         cls, *args: Any, **kwargs: Any
     ) -> Optional[schema.FeatureSettings]:
         raw_capabilities: Any = kwargs.get("raw_capabilities")
+        resource_sku: Any = kwargs.get("resource_sku")
         capabilities: List[SecurityProfileType] = [SecurityProfileType.Standard]
 
         gen_value = raw_capabilities.get("HyperVGenerations", None)
         cvm_value = raw_capabilities.get("ConfidentialComputingType", None)
-        if gen_value and "V2" in str(gen_value):
+        # https://learn.microsoft.com/en-us/azure/virtual-machines/trusted-launch#how-can-i-find-vm-sizes-that-support-trusted-launch # noqa: E501
+        if (
+            gen_value
+            and ("V2" in str(gen_value))
+            and (not raw_capabilities.get("TrustedLaunchDisabled", False))
```

Comment:
Depending the value of `raw_capabilities.get("TrustedLaunchDisabled", False)`. If it's "false", it still need to convert from string to bool. Because `if "false"` returns true.

Reply:
updated